### DISCUSSION TITLE
Rustdesk viewer prevent screen turn off

### DISCUSSION THREAD

**Discussion by oldrichsmejkal** [rustdesk/rustdesk#5921] [General]
### Bug Description

Display (both laptop and external) never goes off when Rustdesk viewer app is open and connected to remote screen.

(When I turn off Ruskdesk viewer, display goes off according to my settings)

### How to Reproduce

Connect to any conputer using Rustdesk, on Windows 11.
Display will be still on.

### Expected Behavior

Display will turn off normally

### Operating system(s) on local side and remote side

Windows 11 -> Ubuntu or also Windows 11

### RustDesk Version(s) on local side and remote side

1.2.2

### Screenshots

non aplicable

**Comment by Unknown78** [comment]
This is still a thing apparently.

I'd like RustDesk to have an option to both enable/enable the prevention of display from turning off.

Even screen saver doesn't work.
